example/cpp/windows/simple_tts/simple_tts.cpp

```diff
@@ -1,6 +1,8 @@
 ﻿// simple_tts.cpp : このファイルには 'main' 関数が含まれています。プログラム実行の開始と終了がそこで行われます。
 //
 
+// TODO: 書き方をunix/song.cppに寄せる
```

Comment:
ファイル名も揃えてあげたみ
こっちをtalk.cppにしていろいろ調整するか、TODOか

```suggestion
// TODO: 書き方をunix/song.cppに寄せる
// TODO: ファイル名をunix/song.cppに寄せる
```